packages/terra-i18n/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "terra-i18n",
+  "main": "lib/I18n.js",
+  "private": true,
+  "version": "0.1.0",
+  "description": "terra-i18n",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-ui.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-i18n",
+    "I18n",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-ui/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-ui#readme",
+  "devDependencies": {
+    "babel-jest": "^19.0.0",
+    "terra-markdown": "^0.1.0",
+    "terra-props-table": "^0.1.0",
+    "terra-toolkit": "^0.1.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "intl": "^1.2.5",
+    "react-intl": "^2.2.3",
```

Comment:
https://github.com/yahoo/react-intl/blob/master/package.json#L106
^ When upgrading react, we should keep this dependency in mind.